src/internal/utils/__tests__/global-flags.test.ts

```diff
@@ -1,17 +1,10 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 import * as globalFlags from '../global-flags';
+import { FlagsHolder, awsuiGlobalFlagsSymbol } from '../global-flags';
 const { getGlobalFlag } = globalFlags;
 
-const awsuiGlobalFlagsSymbol = Symbol.for('awsui-global-flags');
```

Comment:
Some `unique symbol` type madness here. `Symbol.for(key)` returns different symbols even if the key is the same.

So I needed to add an import of `awsuiGlobalFlagsSymbol`. I also could not get it from `const { ... } = globalFlags;` because it also would not be the same `unique symbol`.

But on the upside I managed to remove copies of these interfaces below while keeping everything type safe